packages/sdk/src/node.ts

```diff
@@ -2,8 +2,8 @@ function importNodeModule<T>(name: string): Promise<T> {
   if (!name) {
     throw new Error('import specifier is required');
   }
-  const importModule = new Function('name', `return import(name)`);
-  return importModule(name)
+
+  return import(/* webpackIgnore: true */ name)
```

Comment:
this will cause VM to fail is the import is in new function or eval. Better approach is to diable import transform on our build and tell bundler to treat it as external when possible